website/integrations/services/apple/index.md

```diff
@@ -0,0 +1,385 @@
+This scope mapping will also require an expression to map Apple Business Manager's name field into into separate given and family names. This expression assumes that the name is formatted with the given name first, followed by the family name, delimited by a space. Consider adjusting the expression to match the name format used in your organization.
+
+```py title="Expression field: Extracting given and family names."
+given_name, _, family_name = request.user.name.partition(" ")
+
+return {
+  "given_name": given_name,
+  "family_name": family_name,
+}
+```
+
+Click **Finish** and confirm that new scope mapping is listed in the **Property Mappings** overview.
+
+#### 1.2. Provide read access for the scope mapping
+
+From the **Property Mappings** overview, click **Create**.
+
+When prompted for a choice of mapping, select **Scope Mapping** and continue with the following form values:
+
+| Field      | Value                             |
+| ---------- | --------------------------------- |
+| Name       | `Apple Business Manager ssf.read` |
+| Scope Name | `ssf.read`                        |
+| Expression | `return {}`                       |
+
+Click **Finish** and confirm that new scope mapping is listed in the **Property Mappings** overview.
+
+#### 1.3. Provide management access for the scope mapping
+
+From the **Property Mappings** overview, click **Create**.
+When prompted for a choice of mapping, select **Scope Mapping** and continue with the following form values:
+
+| Field      | Value                               |
+| ---------- | ----------------------------------- |
+| Name       | `Apple Business Manager ssf.manage` |
+| Scope Name | `ssf.manage`                        |
+| Expression | `return {}`                         |
+
+Click **Finish** and confirm that all three new scope mappings are listed in the overview.
+
+### 2. Create signing keys
+
+You will need to create **Signing Key** to sign Security Event Tokens (SET). This key is used to both sign and verify the SETs that are sent between authentik and Apple Business Manager.
+
+From the Admin interface, navigate to **System** -> **Certificates** and clicking **Generate** or **Create**.
+When choosing a **Common Name**, consider using a name that reflects the purpose of the key, such as `apple-business-manager`.
+
+### 3. Create OIDC provider
+
+:::tip Keep your text editor ready
+
+authentik will automatically generate the **Client ID** and **Client Secret** values for the new provider. You'll need these values when configuring Apple Business Manager.
+
+You can always find your provider's generated values by navigating to **Providers**, selecting the provider by name, and clicking the **Edit** button.
+
+:::
+
+From the authentik Admin interface, navigate to **Applications** -> **Providers** and click **Create**.
+In the creation wizard, select **OAuth2/OpenID Provider**. While you can use any name for the provider, we recommend using `Apple Business Manager` for clarity.
```

Comment:
```suggestion
2. For the **Name** of the provider you can use any name; we recommend using `Apple Business Manager` for clarity.

3. For the **Provider Type** select **OAuth2/OpenID Provider**, and then click **Next**.

3. Provide the configuration settings. Be sure to select a Signing Key.

4. Click **Finish** to create and save the provider.
```

Reply:
Do hey need to select a Signing Key here, @GirlBossRush ? Please remove if no...